### PR TITLE
♻️Pydantic V2 upgrade: autoscaling

### PR DESCRIPTION
**Pull request by sanderegg**
## Changes:
- `AUTOSCALING_POLL_INTERVAL` (and any timedelta types) must be defined as ISO8601 strings or integers/floats (e.g. "00:00:10" instead of "10" (this is not accepted anymore!!) **⚠️ DEvops**
- Re-generated the `packages/models-library/src/models_library/generated_models/docker_rest_api.py` using pydanticV2, this will generate some issues in other places, will be fixed later (e.g. all fields are now snake cased)

## How to test

```bash
make devenv
source .venv/bin/activate
cd services/autoscaling
make install-dev
make mypy
make test-dev-unit
```

## Dev-ops checklist
- https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/1003
- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: .env-devel [hunk 1]
```diff
@@ -31,7 +31,7 @@ AUTOSCALING_EC2_ACCESS=null
 AUTOSCALING_EC2_INSTANCES=null
 AUTOSCALING_LOGLEVEL=WARNING
 AUTOSCALING_NODES_MONITORING=null
-AUTOSCALING_POLL_INTERVAL=10
+AUTOSCALING_POLL_INTERVAL="00:00:10"
```
Comment (giancarloromeo) [inline .env-devel:34]: That's more clear. :tada:

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by giancarloromeo — APPROVED** [review]
Thanks a lot.

Just a note: for performance, be sure to not instantiate `TypeAdapter`s in loops/functions continuously called. In that case it's better to put their creation somewhere outside.

See: https://docs.pydantic.dev/latest/concepts/performance/#typeadapter-instantiated-once

**Comment by YuryHrytsuk** [comment]
Is there a related PR in `osparc-config` updating `AUTOSCALING_POLL_INTERVAL`?

**Comment by sanderegg** [comment]
> Is there a related PR in `osparc-config` updating `AUTOSCALING_POLL_INTERVAL`?

@YuryHrytsuk this is coming...

**Pull request merged**